datasets/flwr_datasets/utils.py

```diff
@@ -0,0 +1,32 @@
+"""Utils for FederatedDatasets."""
+from typing import Dict
+
+from flwr_datasets.partitioner import IidPartitioner, Partitioner
+
+
+def _instantiate_partitioners(partitioners: Dict[str, int]) -> Dict[str, Partitioner]:
+    """Transform the partitioners from the initial format to instantiated objects.
+
+    Parameters
+    ----------
+    partitioners: Dict[str, int]
+        Partitioners specified as split to the number of partitions format.
+
+    Returns
+    -------
+    partitioners: Dict[str, Partitioner]
+        Partitioners specified as split to Partitioner object.
+    """
+    instantiated_partitioners: Dict[str, Partitioner] = {}
+    for key, value in partitioners.items():
+        instantiated_partitioners[key] = IidPartitioner(num_partitions=value)
+    return instantiated_partitioners
+
+
+def _check_if_dataset_supported(dataset: str) -> None:
+    """Check if the dataset is in the narrowed down list of the tested datasets."""
+    if dataset not in ["mnist", "cifar10"]:
+        raise ValueError(
+            f"The currently tested and supported dataset are 'mnist' and "
+            f"'cifar10'. Given: {dataset}"
+        )
```

Comment:
Actually, I think we can warn, then it will also support `fashion_mnist`, `food101`, `dog-food`, `zh-plus/tiny-imagenet` (no change needed), so at least I can expand the list.
And a small modification to the I can make it also compatible with `cifar100`, `beans`, cats_vs_dogs`, `keremberke/chest-xray-classification`.
(I went through the most downloaded dataset for Image Classification in the HF Hub)